src/_locales/en-US.json

```diff
@@ -103,6 +103,8 @@
 
   "settings-option-save-page": "Automatically load last viewed page on startup",
 
+  "settings-option-start-minimized": "Start this application minimized",
```

Comment:
Unfortunately this translation key needs to be added as a blank string to the other languages until translators can get to them. Can you update the other files as well?